pkg/clusteragent/admission/mutate/autoinstrumentation/testdata/filter.yaml

```diff
@@ -21,6 +21,17 @@ apm_config:
             language: "java"
         ddTraceVersions:
           java: "v1"
+      - name: "Enabled Prod Namespaces"
+        namespaceSelector:
+          matchLabels:
+            tracing: "yes"
+          matchExpressions:
```

Comment:
Now that I see this, I'm not sure if `matchExpressions` is a good name.
Is it clear that it refers to labels? Maybe this is something that we can discuss later on when we review all the new config options.

Reply:
MatchLabels/MatchExpressions are [a standard Kubernetes concept](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#service-and-replicationcontroller). Straying from that would likely add more confusion.